app/admin.js

```diff
@@ -388,11 +416,17 @@ const run = co(function *(testArgs) {
       handleGenerateHDSeed();
       break;
     case 'verification':
+      requireDB();
       yield handleVerification(args);
       break;
+    case 'recoveryInfo':
+      requireDB();
+      yield handleRecoveryInfo(args);
   }
 
-  db.connection.close();
+  if (db && db.connection) {
+    db.connection.close();
+  }
```

Comment:
we cannot close a db connection if there is no connection. so only close db connection if one exists